### DISCUSSION TITLE
Entity type list missing in Microsoft Search data source

### DISCUSSION THREAD

**Discussion by VerlindenEAM** [microsoft-search/pnp-modern-search#1364] [Ideas]
**Version used**
4.3

**Describe the bug**
When using Microsoft Search as datasource the Person and List entity is now shoing.

**To Reproduce**
1. Go to 'PnP - Search Results' webpart
2. Click on 'Entity types to search'

**Expected behavior**
Entity types to search should match the same entities as described in the Microsoft Search API documentation.
In the Source configuration section of the PnP Modern Search documentation, the entity types to search are pointed to the MS API docs.
https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/#source-configuration_1

**Screenshots**
![image](https://user-images.githubusercontent.com/35732908/134475710-96aba6e3-b77f-49ce-a799-12ebdf2e3d58.png)

**Desktop (please complete the following information):**
 - Browser Edge

**Additional context**
n/a

**Comment by wobba** [comment]
People is not part of the Graph Search API, but we can add "list" for sure.
![image](https://user-images.githubusercontent.com/51104/134495006-1b650839-1414-4318-81dc-8c24ea6ec1bb.png)

**Reply by wobba** [reply]
Added people as well - as it's on the beta endpoint.

**Reply by rajeshb4** [reply]
@wobba Is there any documentation on how to use People entity type from Search Results webpart? I did try using People type and all I got in the results was small person icon without any name or details.

**Reply by wobba** [reply]
@rajeshb4 please ask new questions in new threads. Take a look at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/#people
